### PR TITLE
fix: persist host after signing in

### PR DESCRIPTION
**Pull request by mobitar**

### CODE REVIEW

File: packages/snjs/lib/services/api/session_manager.ts [hunk 1]
```diff
@@ -640,6 +640,7 @@ export class SNSessionManager extends PureService<SessionEvent> {
     const user = data.user;
     this.user = user;
     await this.storageService.setValue(StorageKey.User, user);
+    this.apiService.setHost(this.apiService.getHost());
```
Comment (johnny243) [inline packages/snjs/lib/services/api/session_manager.ts:643]: Why is this needed?

Comment (mobitar) [reply packages/snjs/lib/services/api/session_manager.ts:643]: See here https://app.asana.com/0/1200980603427013/1201146018036636
